Fyber FairBid/AdsScreenViewController.swift

```diff
@@ -235,8 +235,8 @@ extension AdsScreenViewController: FYBRewardedDelegate {
     func rewardedDidShow(_ placementName: String, impressionData: FYBImpressionData) {
         addEventToCallbacksList(#function)
     }
-
-    func rewardedDidFail(toShow placementName: String, withError error: Error) {
+    
```

Comment:
you added some white spaces here

Reply:
sorry i did a tab and Xcode did an indentation.
I removed it in this commit 
1a2d4e4